units/stats_host.go

```diff
@@ -55,8 +55,17 @@ func (j *hostStatsCollector) Run() {
 		return
```

Comment:
Should we grip this error too?

Reply:
The amboy pool will report errors as of the last update, so we don't need to log unless we can add more context here. I think for the stats job, its not useful